test/cmp_msg_test.c

```diff
@@ -283,10 +296,11 @@ static int test_cmp_create_certconf(void)
     fixture->fail_info = 0;
     fixture->expected = 1;
     if (!TEST_true(ossl_cmp_ctx_set0_newCert(fixture->cmp_ctx,
-                                             X509_dup(cert)))) {
+                                             X509_dup_with_libctx(cert)))) {
         tear_down(fixture);
         fixture = NULL;
     }
+    x509_set0_libctx(fixture->cmp_ctx->newCert, libctx, NULL);
```

Comment:
newCert should already be resolved with a libctx here?

Reply:
Oops, this was a leftover. Removed.